blackbox/docs/sql/scalar.txt

```diff
@@ -215,6 +215,41 @@ locale-sensitive or context-sensitive mappings.
     SELECT 1 row in set (... sec)
 
 
+sha1('string') returns string
+-----------------------------
+
+Computes the SHA1 checksum of the given string.
+
+::
+
+    cr> select sha1('crate')
+    ... limit 1;
+    +------------------------------------------+
+    | sha1('crate')                            |
+    +------------------------------------------+
+    | 1673dc397042322a0a5ac49c79cc08d3a25cb0f6 |
+    +------------------------------------------+
+    SELECT 1 row in set (... sec)
+
+sha256('string') returns string
```

Comment:
I'm opting against the `SHA2(x, len)` because we currently only support SHA256. I removed SHA512 because I didn't want to duplicate the upstream `MessageDigests` code. We can add a `SHA2(x, len)` function once we support different SHA2 sizes.